### PR TITLE
update ediutilities dependencies and fix errors

### PR DESCRIPTION
**Pull request by joannekoch**

### CODE REVIEW

File: nutrient-transect-eml-assembly.Rmd [hunk 1]
```diff
@@ -13,23 +13,24 @@ knitr::opts_chunk$set(message = FALSE, warning = FALSE)
...
+library(ediutilities)
+library(here)
+library(lubridate)
+library(tidyverse)
+require(readxl)
+require(lubridate)
+require(devtools)
+require(EML)
+require(maps)
+require(xml2)
```
Comment (joefutrelle) [inline nutrient-transect-eml-assembly.Rmd:26]: do we want all of these to be `library`?

File: nutrient-transect-eml-assembly.Rmd [hunk 2]
```diff
@@ -186,10 +192,7 @@ edi_filename <- "nes-lter-nutrient-transect"
 pkg_id <- "knb-lter-nes.4.2"
```
Comment (joefutrelle) [inline nutrient-transect-eml-assembly.Rmd:192]: bump to 4.3

**Comment by joannekoch** [comment]
1. This line was changed to display error in terminal window if the cruise was not found:
cruise_nut <- tryCatch(read_csv(nut_url),
+                  error=function(cond) 
+                      { message(cond)
+                        return(NA) })

Error Displayed: HTTP error 500.https://nes-lter-data.whoi.edu/api/nut/ar22.csv

2. Added quote=FALSE to write.csv(all_cruises, "nes-lter-nutrient-transect.csv", row.names = FALSE, quote = FALSE)
to write a data frame as a csv file without quoting values and strings. The quote is used in other packages.

@joefutrelle - .Rmd file changes made and commited to pull request

**Comment by joannekoch** [comment]
@joefutrelle , good catch -  knb-lter-nes.4.2.xml deleted

**Pull request merged**